README.md

```diff
@@ -22,7 +22,7 @@
 
          git clone https://github.com/Oeffis/Oeffis.git
 
-2.  Install NPM packages in Oeffis/frontend and Oeffis/backend:
+2. Install NPM packages in Oeffis/vrr_client, Oeffis/backend and Oeffis/frontend (sticking to this order):
```

Comment:
the "sticking to this order" info here is good, but I think we should also add the info, why this is necessary. Maybe this isn't the case anymore in the future and then we wouldn't know wether we can remove this notice.